data/json/items/generic/bathroom_house.json

```
@@ -233,5 +233,102 @@
     "material": [ "steel", "plastic" ],
     "bashing": 3,
     "to_hit": 1
+  },
+  {
+    "id": "comb_lice",
+    "name": { "str": "lice comb" },
+    "type": "GENERIC",
+    "description": "A stainless-steel comb with spiral grooved teeth designed to remove pesky parasites from hair.  Even just thinking about it makes you itch.",
+    "symbol": "c",
+    "material": [ "steel" ],
+    "copy-from": "base_toiletries"
+  },
+  {
+    "id": "bath_toy",
+    "name": { "str": "bath toy" },
+    "type": "GENERIC",
+    "description": "A plastic floating bath toy.",
+    "symbol": "w",
+    "color": "yellow",
+    "material": [ "rubber" ],
+    "copy-from": "base_toiletries",
+    "weight": "55 mg",
+    "volume": "53 ml",
+    "snippet_category": [
+      { "id": "btoy1", "text": "A plastic floating bath toy." },
+      { "id": "btoy2", "text": "A yellow rubber duck.  Designed to dropped in the bath." },
+      { "id": "btoy3", "text": "A black rubber duck with large white eyes.  Designed to dropped in the bath." },
+      { "id": "btoy4", "text": "A small green dinosaur with a wide grin.  Designed for bath time." }
+    ]
+  },
+  {
+    "id": "toothbrush_electric",
+    "name": { "str": "electric toothbrush" },
+    "type": "GENERIC",
+    "description": "A plastic electric toothbrush.",
+    "symbol": "/",
+    "material": [ "plastic" ],
+    "copy-from": "base_toiletries",
+    "weight": "140 g",
+    "volume": "107 ml",
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "rigid": true,
+        "flag_restriction": [ "BATTERY_LIGHT", "BATTERY_ULTRA_LIGHT" ],
+        "default_magazine": "light_battery_cell"
+      }
+    ]
+  },
+  {
+    "id": "bottle_bathroom",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "plastic bottle" },
```

```suggestion
    "name": { "str": "liquid soap bottle" },
```
To avoid confusion with normal plastic bottles.

---

they are functionally the same when empty which is why I left it as a plastic bottle.  I'm not adverse to this change completely but I think there is maybe merit in just leaving the name as 'plastic bottle'. I'd like to see how others think, too.